packages/server/tests/acceptance/erc20.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('@erc20 Acceptance Tests', async function () {
         recipient = accounts[1].address;
         anotherAccount = accounts[2].address;
 
-        // alow mirror node a 2 full record stream write windows (2 sec) and a buffer to persist setup details
+        // allow mirror node a 5 full record stream write windows (5 sec) and a buffer to persist setup details
```

Comment:
```suggestion
        // allow mirror node 5 full record stream write windows (2 secs each) and a buffer to persist setup details
```

Reply:
Also if the comment is true then the timeout would become 11 secs.
That is each window is 2 secs. 5 windows is 10 secs with a 1 sec buffer.
Is the intent still applicable and the effect still desired?

Reply:
Hmm, i've copied this comment around the code, but basically we are setting a 5 second sleep here, nothing more. I've noticed that now it needs around 4-5 seconds for the changes to reach web3 module. Earlier we were asking the consensus node, but now that we are getting this from the mirror node it has a delay.